### PR TITLE
Misc revisions and shortenings

### PR DESCRIPTION
**Pull request by benjub**
See commit messages.

@avekens : I made a small shortening in your mathbox (rexrsb); I deemed it not necessary to put a revision tag if that's ok with you.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -42472,6 +42448,71 @@ Equality axioms (and Extensionality) yield
...
+       of ~ sspwb since it requires fewer axioms.  (Revised by BJ,
+       13-Apr-2024.) $)
+    sspw $p |- ( A C_ B -> ~P A C_ ~P B ) $=
+      ( vx wss cpw cv wcel sstr2 com12 velpw 3imtr4g ssrdv ) ABDZCAEZBEZMCFZADZ
+      PBDZPNGPOGQMRPABHICAJCBJKL $.
+  $}
+
+  ${
+    sspwi.1 $e |- A C_ B $.
+    $( The powerclass perserves inclusion (inference form).  (Contributed by
```
Comment (icecream17) [inline set.mm:42464]:
```suggestion
    $( The powerclass preserves inclusion (inference form).  (Contributed by
```

File: set.mm [hunk 2]
```diff
@@ -42472,6 +42448,71 @@ Equality axioms (and Extensionality) yield
...
+       BJ, 13-Apr-2024.) $)
+    sspwi $p |- ~P A C_ ~P B $=
+      ( wss cpw sspw ax-mp ) ABDAEBEDCABFG $.
+  $}
+
+  ${
+    sspwd.1 $e |- ( ph -> A C_ B ) $.
+    $( The powerclass perserves inclusion (deduction form).  (Contributed by
```
Comment (icecream17) [inline set.mm:42472]:
```suggestion
    $( The powerclass preserves inclusion (deduction form).  (Contributed by
```

**Review by icecream17 — APPROVED** [review]

File: set.mm (hunk 1 above)
Comment (benjub) [reply set.mm:42464]: Fixed in https://github.com/metamath/set.mm/pull/3918/commits/1d884fa9eef7707551df5107b59fa74d2bc0c268.

File: set.mm (hunk 2 above)
Comment (benjub) [reply set.mm:42472]: Fixed in https://github.com/metamath/set.mm/pull/3918/commits/1d884fa9eef7707551df5107b59fa74d2bc0c268.

**Review by wlammen — APPROVED** [review]
Assuming @avekens consent.

**Review by tirix — APPROVED** [review]

**Comment by wlammen** [comment]
Obsoleted by #3933 and #3934

**Comment by benjub** [comment]
Closing as subsumed by #3933.

**Pull request closed without merging**